### DISCUSSION TITLE
Add Support for Realms

### DISCUSSION THREAD

**Discussion by nooneisback** [ConcreteMC/Alex#115] [Feature requests]
Since realms don't allow modding beyond data packs, they are a great use case for this project. It is currently impossible to connect to realms as they use a hidden dynamic IP and don't support direct connection.

**Comment by kennyvv** [comment]
This would be cool to have, i'll have to think about it.

**Reply by kennyvv** [reply]
Realms API Documentation:
Bedrock: https://lukeeey.github.io/mcpe-docs/rest-api/
Java: https://wiki.vg/Realms_API
